src/Transforms/Repeater.php

```diff
@@ -0,0 +1,62 @@
+<?php namespace Leean\Acf\Transforms;
+
+/**
+ * Class Repeater.
+ *
+ * @package Leean\Acf\Transforms
+ */
+class Repeater
+{
+	/**
+	 * Apply the transforms
+	 *
+	 * @param array $field The field.
+	 * @return array
+	 */
+	public static function apply( $field ) {
```

Comment:
I think is very similar to the function above, can we use only one function. I think having an abstract class called `Transform`  might help with the task.